packages/svelteui-core/src/components/Modal/Modal.svelte

```diff
@@ -49,13 +49,16 @@
 	const bodyId = `${baseId}-body`;
 	const _overlayOpacity =
 		typeof overlayOpacity === 'number' ? overlayOpacity : $colorScheme === 'dark' ? 0.85 : 0.75;
+	const { handleFocusReturn } = useFocusReturn();
 
 	const closeOnEscapePress = (event: KeyboardEvent) => {
 		if (!trapFocus && event.code === 'Escape' && closeOnEscape) {
 			onClose();
 		}
 	};
 
+	$: handleFocusReturn(opened);
```

Comment:
nitpicking, but can you move this below, next to the all the other reactive statements?